### DISCUSSION TITLE
How to disable qflist preview transparency?

### DISCUSSION THREAD

**Discussion by zer09** [catppuccin/nvim#572] [Q&A] [answered]
![image](https://github.com/catppuccin/nvim/assets/780780/b09cf21b-6d6f-4d5f-84b9-a3d374edc9c9)

example above qflist preview, how I can disable that transparency?

Thank you

**Comment by rewhile** [comment, marked as answer]
```lua
{
  "kevinhwang91/nvim-bqf",
  opts = {
    preview = { winblend = 0 },
  },
},
```

For more details please see [kevinhwang91/nvim-bqf](https://github.com/kevinhwang91/nvim-bqf#setup-and-description)

**Reply by zer09** [reply]
Hello, I'm sorry to bother you, but I tried to set both to 0 but it doesn't have any effect

**Reply by rewhile** [reply]
See my edited answer, does it work for you?

![image](https://github.com/catppuccin/nvim/assets/56817415/f9961a45-49bd-4ca5-b2a3-5e17a4eeff44)

**Reply by zer09** [reply]
Thank you so much, it works now. I never thought it was nvim-bqf setting the transparency
